src/py/pyproject.toml

```
@@ -26,7 +26,7 @@ maintainers = [
   {name = "Andrew Pikul", email = "ajpikul@gmail.com"},
 ]
 dependencies = [
-  "choreographer>=1.1.1",
+  "choreographer>=1.3.0rc2",
```

```suggestion
  "choreographer>=1.2.1",
```
I'd suggest to loosen this pin unless Kaleido absolutely needs the 1.3.0 RC